paddle/rockcraft.yaml

```diff
@@ -0,0 +1,64 @@
+# Based on https://github.com/kserve/kserve/blob/release-0.11/python/paddle.Dockerfile
+name: paddle-server
+summary: Paddle server for Kserve deployments
+description: "Kserve Paddle server"
+version: "0.11.0"
+license: Apache-2.0
+base: ubuntu@22.04
+platforms:
+    amd64:
+run-user: _daemon_
+services:
+  paddle-server:
+    override: replace
+    summary: "Paddle server service"
+    startup: enabled
+    command: "python3.10 -m paddleserver [ ]"
```

Comment:
I feel like there's something we can do in staging the rock to make this work with just upstream's `python -m paddleserver` so we don't need to include the version

One way might just be making a symbolic link, but I wonder if there is anything better

Reply:
I dont think this adds any value tbh :) lets skip